### PR TITLE
[MRG] Changing `sourmash compute` to `sourmash sketch` in test files

### LINKED ISSUE #1710: Changing `sourmash compute` to `sourmash sketch` in test files
See #1419 for reference.

In #1536 and #1680, `sourmash compute` has been replaced with `sourmash sketch` in `tests/test_sourmash.py`.

In this issue, it is to be replaced in:
- `tests/test_cmd_signature.py`
- `tests/test_lca.py`

### PR DESCRIPTION
**Pull request by keyabarve**
Fixes #1710

Replace `compute` with `sketch` in:
- `tests/test_cmd_signature.py`
- `tests/test_lca.py`

NOTE: One test: `tests/test_cmd_signature.py::test_sig_describe_1_hp` can't be changed from `compute` to `sketch`.

### CODE REVIEW

**Comment by keyabarve** [comment]
@ctb One test from `tests/test_cmd_signature.py` is failing. This is the error message:
```
>           raise SourmashCommandFailed(self.last_result.err)

E           == This is sourmash version 4.1.2.dev6+g87a908c.d20210609. ==
== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==
E           
E           Traceback (most recent call last):
E             File "/Users/keya/sourmash/tests/sourmash_tst_utils.py", line 105, in runscript
E               status = _runscript(scriptname)
E             File "/Users/keya/sourmash/tests/sourmash_tst_utils.py", line 47, in _runscript
E               pkg_resources.load_entry_point("sourmash", 'console_scripts', scriptname)()
E             File "/Users/keya/sourmash/src/sourmash/__main__.py", line 13, in main
E               return mainmethod(args)
E             File "/Users/keya/sourmash/src/sourmash/cli/sketch/translate.py", line 97, in main
E               return sourmash.command_sketch.translate(args)
E             File "/Users/keya/sourmash/src/sourmash/command_sketch.py", line 259, in translate
E               raise ValueError("cannot set both --dayhoff and --hp")
E           ValueError: cannot set both --dayhoff and --hp

tests/sourmash_tst_utils.py:195: SourmashCommandFailed
```
I can't seem to figure out how to fix this.

**Comment by ctb** [comment]
it would be helpful to know the test name...

in re
>ValueError: cannot set both --dayhoff and --hp

I don't think you can create signatures like this with sourmash sketch,
so just note that this test can't be updated to use sketch in the PR
description, and we'll create a new issue to tackle it when we merge
the PR.

**Comment by keyabarve** [comment]
> it would be helpful to know the test name... in re
> ValueError: cannot set both --dayhoff and --hp
>
This is the test: `tests/test_cmd_signature.py::test_sig_describe_1_hp`

> I don't think you can create signatures like this with sourmash sketch, so just note that this test can't be updated to use sketch in the PR description, and we'll create a new issue to tackle it when we merge the PR.
>
Alright!

**Comment by keyabarve** [comment]
@ctb Ready for review and merge!

File: tests/test_cmd_signature.py [hunk 1]
```diff
@@ -2587,7 +2587,7 @@ def test_sig_describe_1_hp(c):
     c.run_sourmash('compute', '-k', '21,30',
                    '--dayhoff', '--hp', '--protein',
                    '--dna',
-                   testdata)
+                   testdata)
```
Comment (ctb) [inline tests/test_cmd_signature.py]: unintentional change

**Review by ctb — CHANGES_REQUESTED** [review]
looks good! just one thing to change.

please also change title to MRG

**Comment by keyabarve** [comment]
@ctb Ready for review and merge!

**Review by ctb — APPROVED** [review]

**Comment by keyabarve** [comment]
Should I merge this?

**Comment by ctb** [comment]
sure!

**Pull request merged**